README.md

```diff
@@ -106,21 +106,21 @@ c6gn.16xlarge
 c6in.16xlarge
 c7gn.8xlarge
 dl1.24xlarge
+f2.48xlarge
```

Comment:
qq: when do we usually update these instances, how frequently we need to update them? whenever we have new launches we need to include them in this code base? same with the prices below, how frequently we do update because I haven't seen these details been updated for a while..

Reply:
I only update them when I plan on doing a release. It's only demo stuff for the README. In real operation these are all pulled dynamically.